tests/php/src/Validation/ScannableURLsRestControllerTest.php

```diff
@@ -77,6 +79,9 @@ public function test_get_items_permissions_check() {
 	 * @covers ::get_item_schema()
 	 */
 	public function test_get_items() {
+		wp_set_current_user( self::factory()->user->create( [ 'role' => 'administrator' ] ) ); // Because the setting was registered and not cleaned up?
+		AMP_Options_Manager::update_option( Option::READER_THEME, ReaderThemes::DEFAULT_READER_THEME );
+		AMP_Options_Manager::update_option( Option::THEME_SUPPORT, AMP_Theme_Support::READER_MODE_SLUG );
```

Comment:
I'm not sure why the options need to be set now where they didn't before. Again, seems to indicate the database not being reset. Also, the need to set the current user indicates the setting was registered and thus the capability check is being done.